impeller/aiks/picture.h

```diff
@@ -8,13 +8,17 @@
 #include <memory>
 
 #include "flutter/fml/macros.h"
+#include "impeller/aiks/aiks_context.h"
+#include "impeller/aiks/image.h"
 #include "impeller/entity/entity.h"
 #include "impeller/entity/entity_pass.h"
 
 namespace impeller {
 
 struct Picture {
   std::unique_ptr<EntityPass> pass;
+
+  std::shared_ptr<Image> RenderToImage(AiksContext& renderer);
```

Comment:
Nit: `AiksContext& context`?

Reply:
Another idea could be to flip it and ask the AiksContext to snapshot the picture as an image.

Reply:
Done.